service/src/main/java/uk/nhs/adaptors/gp2gp/common/service/ProcessFailureHandlingService.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.adaptors.gp2gp.common.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.adaptors.gp2gp.ehr.EhrExtractStatusRepository;
+import uk.nhs.adaptors.gp2gp.ehr.EhrExtractStatusService;
+import uk.nhs.adaptors.gp2gp.ehr.SendNegativeAcknowledgementTaskDispatcher;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ProcessFailureHandlingService {
```

Comment:
Not sure if I should use `conversation` instead of `process`. Thoughts?

Reply:
I would stay with process.